applications/asset_tracker/prj_nrf9160_pca20035ns.conf

```diff
@@ -121,3 +122,4 @@ CONFIG_DOWNLOAD_CLIENT_STACK_SIZE=4096
 CONFIG_RESET_ON_FATAL_ERROR=n
 
 CONFIG_WATCHDOG=y
+CONFIG_MQTT_KEEPALIVE=120
```

Comment:
Is MQTT_KEEPALIVE change intentional? If so, would be good to mention in the commit message how it's related.

Reply:
In some networks with shortlived NAT rules the 20 minute default timeout causes trouble. Agree that the reason can be mentioned in the commit message.

Reply:
I was debating whether to include it.  I could just remove it, and then do my other PR that also fixes keepalive timing strangeness.  Maybe I'll do that.